library/Kubernetes/Web/Factory.php

```diff
@@ -27,16 +27,23 @@
 use ipl\Html\ValidHtml;
 use ipl\Orm\Model;
 use ipl\Orm\Query;
-use ipl\Stdlib\Filter\Rule;
+use ipl\Sql\Connection;
 use ipl\Web\Url;
-use ipl\Web\Widget\EmptyState;
 use ipl\Web\Widget\Icon;
 
 abstract class Factory
```

Comment:
In addition to the requested changes this class, please use `self::canonicalizeKind($kind)` everywhere, omit comma in the last match case and use direct return with match where appropriate.